crates/transaction-pool/src/pool/txpool.rs

```diff
@@ -245,6 +245,39 @@ impl<T: TransactionOrdering> TxPool<T> {
         }
     }
 
+    /// Returns an iterator that yields transactions that are ready to be included in the block with
+    /// the given base fee and optional blob fee.
+    pub(crate) fn best_transactions_with_attributes(
```

Comment:
Hey @mattsse  please let me know your opinion about this function and then I can also write the related unit tests